### PR TITLE
sleep in script

### PR DESCRIPTION
**Pull request by nsluhrs**
added a script that allows players to skip to night by right clicking a bed

### CODE REVIEW

File: programs/survival/sleep_in.sc [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  )
+);
+
+_skip_day() -> (
+	//dtime = day_time()%24000; //day time
+	if((dtime = day_time()%24000)< global_night_time, // if it's sleepy time
+		current = day_time();
+		day_time(current + (global_night_time - dtime) );
+		print('day skipped')
+	);
```
Comment (Firigion) [inline programs/survival/sleep_in.sc:33]: This dpoesn't work if it's already night. Again, if that's intended, disregard this, but I'm not sure that's intended.

**Comment by Firigion** [comment]
Ah, also: think about making the message  abit more visible. Could be a message over the hotbar (I forge tthe name), a title screen or a coloured chat message.

File: programs/survival/sleep_in.sc (hunk 1 above)
Comment (Ghoulboy78) [reply programs/survival/sleep_in.sc:33]: well I think it is intended, seen as though this script is trying to skip the day. If it's night, then there is no need to skip the day.

**Review by Ghoulboy78 — APPROVED** [review]
Looks pretty good to me

**Comment by nsluhrs** [comment]
Closed this by mistake but as long as I'm remaking it will make change recommended by Firigion then resubmit

**Pull request closed without merging**